loyalty_initial_date_validity/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+To use this module simply go to edit or create a promotion and set a start date in
+the field labelled "Start date".
```

Comment:
Well, point to the sale_... extension as the field here is useless on its own

Reply:
done!